outputs/costs_2020.csv

```diff
@@ -669,9 +674,13 @@ central gas boiler,VOM,1.164,EUR/MWh_th,"Danish Energy Agency, technology_data_f
 central gas boiler,efficiency,1.03,per unit,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx","44 Natural Gas DH Only:  Total efficiency , net, annual average",2015.0
 central gas boiler,investment,63.4933,EUR/kW_th,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx",44 Natural Gas DH Only:  Nominal investment,2015.0
 central gas boiler,lifetime,25.0,years,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx",44 Natural Gas DH Only:  Technical lifetime,2015.0
-central geothermal-sourced heat pump,FOM,0.8328,%/year,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx","45.1.a Geothermal DH, 1200m, E:  Fixed O&M",2015.0
+central geothermal heat source,FOM,1.4715,%/year,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx","45.1.a Geothermal DH, 1200m, E:  Fixed O&M",2015.0
+central geothermal heat source,VOM,6.0281,EUR/MWh,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx","45.1.a Geothermal DH, 1200m, E:  Variable O&M",2015.0
+central geothermal heat source,investment,1625.2908,EUR/kW,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx","45.1.a Geothermal DH, 1200m, E:  Nominal investment",2015.0
+central geothermal heat source,lifetime,25.0,years,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx","45.1.a Geothermal DH, 1200m, E:  Technical lifetime",2015.0
+central geothermal-sourced heat pump,FOM,3.3935,%/year,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx","45.1.a Geothermal DH, 1200m, E:  Fixed O&M",2015.0
 central geothermal-sourced heat pump,VOM,6.0281,EUR/MWh_th,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx","45.1.a Geothermal DH, 1200m, E:  Variable O&M",2015.0
-central geothermal-sourced heat pump,investment,2871.7979,EUR/kW_th,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx","45.1.a Geothermal DH, 1200m, E:  Nominal investment",2015.0
+central geothermal-sourced heat pump,investment,704.7435,EUR/kW_th,"Danish Energy Agency, technology_data_for_el_and_dh.xlsx","45.1.a Geothermal DH, 1200m, E:  Nominal investment",2015.0
```

Comment:
That requires a warning in the release note that cost components are pulled out of this item.

Reply:
Thanks for the review! I've added the warning.